directives/run-async.js

```diff
@@ -53,6 +53,11 @@ export const runAsync = directive((key, task, templates) => (part) => {
 		const pendingPromise = new Promise((res, rej) => {
 			resolvePending = res;
 			rejectPending = rej;
+		}).catch((error) => {
```

Comment:
The `runAsync` directive handles the `InitialStateError` by displaying the initial template.  As such, it should catch such error so that it does not pollute the console.